.changeset/late-planets-rescue.md

```diff
@@ -0,0 +1,2 @@
+---
```

Comment:
Add a description to this change as it touches on the SDK and adds a new method.

It could be: `Add to get selected account from wallet via SDK`